dartagnan/src/main/java/com/dat3m/dartagnan/expression/op/IOpBin.java

```diff
@@ -39,7 +39,7 @@ public String toString() {
         }
     }
 
-    public String toLinuxName(){
+    public String toTextualName(){
```

Comment:
The naming is odd: What would be a "non-textual" name?
I think we should just call it `getName` and return the name of the enum value in lower case.
And we should rename the enum values `PLUS` to `ADD`, `MINUS` to `SUB` and `MULT` to `MUL`, and remove the underscores from `L_SHIFT` and the other enum names.
Then you could just do `return this.name().toLowerCase()` and it works for all op codes (it's weird to have undefined names for some enum values in the first place).

On a side note: I think it would be most natural to have `toString` return the name and add a `getSymbol` method to get the operator symbol. But this is a bigger change since all the callers of `op.toString` would need to get updated, so it should be done in a separate PR.

Reply:
You didn't rename `PLUS`/`MINUS`. This will cause some events to be named like `atomic_fetch_plus` rather than `atomic_fetch_add`.